### DISCUSSION TITLE
I want to remove the right panel completely

### DISCUSSION THREAD

**Discussion by ghost** [cotes2020/jekyll-theme-chirpy#1829] [Q&A] [answered]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `chirpy-starter`

### Description

I want to remove the right panel completely, and make my site look the same from 1000px to 1920px wide devices without the right panel.

### Operations you have already tried

I could delete the tags and latest posts from the panel, but cannot remove the panel itself.
I tried setting 
```
.panel-wrapper {
  display: none !important;
}
```
in the `assets/css/jekyll-theme-chirpy.scss`
and also some other things over the past 5 hours, nothing works. i'm not that good at programming so tried to take help from ai, but everything failed.

**Comment by kungfux** [comment, marked as answer]
To hide the panel, you must use the `ID` qualifier instead of the `class` qualifier:

```css
#panel-wrapper {
  display: none !important;
}
```

However, this will not solve the problem completely, as the main content panel will not use the available space.

The easiest way would be to override the `default` layout by creating `_layouts/default.html` with custom content to remove the panel and customize the bootstrap classes, e.g.

```html
---
layout: compress
---
<!doctype html>

{% include origin-type.html %}

{% include lang.html %}

{% if site.theme_mode %}
  {% capture prefer_mode %}data-mode="{{ site.theme_mode }}"{% endcapture %}
{% endif %}

<!-- `site.alt_lang` can specify a language different from the UI -->
<html lang="{{ page.lang | default: site.alt_lang | default: site.lang }}" {{ prefer_mode }}>
  {% include head.html %}

  <body>
    {% include sidebar.html lang=lang %}

    <div id="main-wrapper" class="d-flex justify-content-center">
      <div class="container d-flex flex-column px-xxl-5">
        {% include topbar.html lang=lang %}

        <div class="row flex-grow-1">
          <main aria-label="Main Content" class="col-12">
            {% if layout.refactor or layout.layout == 'default' %}
              {% include refactor-content.html content=content lang=lang %}
            {% else %}
              {{ content }}
            {% endif %}
          </main>
        </div>

        <div class="row">
          <!-- tail -->
          <div id="tail-wrapper" class="col-12">
            {% for _include in layout.tail_includes %}
              {% assign _include_path = _include | append: '.html' %}
              {% include {{ _include_path }} lang=lang %}
            {% endfor %}

            {% include_cached footer.html lang=lang %}
          </div>
        </div>

        {% include_cached search-results.html lang=lang %}
      </div>

      <aside aria-label="Scroll to Top">
        <button id="back-to-top" type="button" class="btn btn-lg btn-box-shadow">
          <i class="fas fa-angle-up"></i>
        </button>
      </aside>
    </div>

    <div id="mask"></div>

    {% if site.pwa.enabled %}
      {% include_cached notification.html lang=lang %}
    {% endif %}

    <!-- JavaScripts -->
    {% include js-selector.html lang=lang %}

    {% include_cached search-loader.html lang=lang %}
  </body>
</html>
```

This will give you the closest example to what you need, but may require some fine-tuning.

P.S. You may find the original content of `default` layout [here](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/_layouts/default.html).

**Reply by ghost** [reply]
I did try something similar before, didn't seem to work.
Let me try this I'll let you know.
Thank you for your time ❤️

**Reply by ghost** [reply]
That looks exactly like what I wanted, you're a life saver.
Have a great day. <3

**Reply by kungfux** [reply]
> That looks exactly like what I wanted, you're a life saver.
> Have a great day. <3

It's good to know 😁 
Thanks. You too. 👍
